app/models/section_home.rb

```diff
@@ -3,4 +3,7 @@ class SectionHome < Node
   validates :parent, ancestry_depth: { equals: 1 }
   validates :section, presence: true, uniqueness: true
 
+  def path_elements
+    [section.home_node.slug]
+  end
```

Comment:
Do section home nodes actually need an override here? I thought they were working fine without it.